test/exoffice_test.exs

```diff
@@ -33,5 +33,38 @@ defmodule ExofficeTest do
       assert is_pid(pid)
       assert Exoffice.get_rows(pid, parser) |> Enum.to_list() == expected
     end
+
+    test "decode CSV in a safe way" do
+      [{:ok, pid, parser}] =
+        Exoffice.parse("./test/test_data/test_semicolon.csv", parser_options: [safe: true, separator: ?;])
```

Comment:
Thank you for your PR, could you please add a test with invalid CSV data to make sure it will not raise in safe mode?

Reply:
Sure! Done :)